src/buildtool.ts

```diff
@@ -7,7 +7,7 @@ export interface RunBuildOptions {
 }
 
 export function generateCommand(options: RunBuildOptions): string {
-    const pluginsPath = path.join(__dirname,"plugins").replace("'","''");
+    const pluginsPath = path.join(__dirname,"plugins").replace(/'/g,"''");
```

Comment:
```suggestion
    const pluginsPath = path.join(__dirname,"plugins").replaceAll("'","''");
```
`replaceAll` might be clearer?